Tallies/TallyMaps/Maps1D/directionMap_class.f90

```diff
@@ -0,0 +1,194 @@
+module directionMap_class
```

Comment:
No worries, I understood! It's a bit of a long name. I personally would leave it as it is until a polarMap comes by. Also because you can map direction on any plane with this, so it's quite general.

Reply:
In this case, it might be even more convenient to leave it generic and add the possibility to modify the angle range to anything within -PI to PI. So one could tally -PI to PI on a plane and 0 to PI on another

Reply:
I suppose it would still be a fraction of the azimuthal plane. Wouldn't the polar equivalent need some weighting by a sine? But, in principle, the range could be limited. I imagine that might make things a bit messier and also wouldn't be requested very frequently so I wouldn't worry much about it.

One more thing: which direction would the 0 angle correspond to in a given plane? Or pi/2? It would be good to mention these explicitly at least in the comments.

Reply:
I think that if you want to integrate over a polar angle you need to add the sign weighting as a Response, rather than in the tally Map. Now I added a custom min and max angle, so that it generalises better.